.gitignore

```diff
@@ -36,3 +37,9 @@
 /.project
 /.pydevproject
 /.settings/
+
+# For Performance Team
+*.csv
+*.log
+*.swp
```

Comment:
This is for vim, right? Please place this in your global gitignore file. You can do so by placing something like this in your `~/.gitignore`:

    [core]
        excludesfile = /home/ichimonji10/.gitignore_global

And something like this in your `~/.gitignore_global`:

    # Vim swapfiles
    *.swp

Yes, I do understand that we ignore eclipse-specific files. But that is a concession made simply for the sake of moving things along. If you're a vim user, I expect you're also competent enough to edit your `~/.git*` files.